release-notes/README.md

```diff
@@ -44,12 +44,20 @@ For more details, see the documentation and release notes in the [aepsdk-react-n
 ### iOS AEPEdge 1.4.0
 
 * Updates timestamp in Experience Events to use fractional seconds.
-* Deprecates APIs `XDMFormatters.dateToISO8601String` and `XDMFormatters.dateToFullDateString`. Use the `Date` extension methods `getISO8601UTCDateWithMilliseconds` and `getISO8601FullDate` instead, provided by the AEPServices module within the AEPCore extension. 
+* Deprecates APIs `XDMFormatters.dateToISO8601String` and `XDMFormatters.dateToFullDateString`. Use the `Date` extension methods `getISO8601UTCDateWithMilliseconds` and `getISO8601FullDate` instead, provided by the AEPServices module within the AEPCore extension.
 
 ### iOS AEPEdgeConsent 1.0.1
 
 * Updates timestamp in Consent requests to use fractional seconds.
 
+## April 2, 2022
+
+### iOS AEPAssurance 1.1.4
+
+* Fixed a bug that caused Assurance to not connect to a session if your iOS app's info.plist contains property of type date.
+
+Note: This release pertains to Assurance mobile extension that works with ACPCore
```

Comment:
I'm not sure what this note means.

Reply:
@calebk1 
Unlike other iOS extensions where
ACPAnalytics works with ACPCore
and the new swift based 
AEPAnalytics works with AEPCore

Assurance is a little unusual where we have the same name AEPAssurance extension  but

AEPAssurance 1.x.x works with ACPCore
AEPAssuracne 3.x.x works with AEPCore

So this release is an update to AEPAssurance that works with ACPCore